metadata.rb

```diff
@@ -5,8 +5,10 @@
 description      'Installs/Configures magento-ng'
 long_description 'Installs/Configures magento-ng'
 version          '0.4.1'
+source_url       'https://github.com/inviqa/chef-magento-ng'
+issues_url       'https://github.com/inviqa/chef-magento-ng/issues'
 
 depends 'config-driven-helper', '~> 1.5'
-depends 'cron', '~> 1.4'
+depends 'cron', '~> 1.7'
```

Comment:
no need to bump this version. 1.7 included in ~> 1.4, and lowest minimum version supported best. This including all between 1.4 and < 2.0